packages/manager/apps/catalog-revamp/src/components/Filters/Filters.tsx

```diff
@@ -0,0 +1,177 @@
+            <div className="filters-universes">
+              <OsdsText
+                level={OdsTextLevel.heading}
+                size={OdsTextSize._400}
+                className="title"
+              >
+                {t('manager_catalog_filters_universes')}
+              </OsdsText>
+              {universes.map((item: string, id: number) => (
+                <span key={`universe${id}`}>
+                  <OsdsCheckbox>
+                    <OsdsCheckboxButton
+                      size={OdsCheckboxButtonSize.sm}
+                      color={OdsThemeColorIntent.primary}
+                    >
+                      <span slot="end">{item}</span>
+                    </OsdsCheckboxButton>
+                  </OsdsCheckbox>
+                </span>
+              ))}
+              {!universes.length &&
+                [...Array(5)].map((item: string, id: number) => (
+                  <div key={`universe${id}`}>
+                    <OsdsCheckbox>
+                      <OsdsCheckboxButton
+                        size={OdsCheckboxButtonSize.sm}
+                        color={OdsThemeColorIntent.primary}
+                      >
+                        <span slot="end">
+                          <OsdsSkeleton />
+                        </span>
+                      </OsdsCheckboxButton>
+                    </OsdsCheckbox>
+                  </div>
+                ))}
+            </div>
+            <div className="filters-categories">
+              <OsdsText
+                level={OdsTextLevel.heading}
+                size={OdsTextSize._400}
+                className="title"
+              >
+                {t('manager_catalog_filters_categories')}
+              </OsdsText>
+              <div className="filters-categories-content">
+                {categories.map((item: string, id: number) => (
+                  <div key={`category${id}`}>
+                    <OsdsCheckbox>
+                      <OsdsCheckboxButton
+                        size={OdsCheckboxButtonSize.sm}
+                        color={OdsThemeColorIntent.primary}
+                      >
+                        <span slot="end">{item}</span>
+                      </OsdsCheckboxButton>
+                    </OsdsCheckbox>
+                  </div>
+                ))}
+                {!categories.length &&
+                  [...Array(15)].map((item: string, id: number) => (
+                    <div key={`category${id}`}>
+                      <OsdsCheckbox>
+                        <OsdsCheckboxButton
+                          size={OdsCheckboxButtonSize.sm}
+                          color={OdsThemeColorIntent.primary}
+                        >
+                          <span slot="end">
+                            <OsdsSkeleton />
+                          </span>
+                        </OsdsCheckboxButton>
+                      </OsdsCheckbox>
+                    </div>
+                  ))}
```

Comment:
this is the same component as above : avoid duplicating lines like this when you can do a component

Reply:
Done FilterItem component created